src/components/SearchFilter.scss

```diff
@@ -0,0 +1,21 @@
+@import "~@blueprintjs/core/dist/variables";
+
```

Comment:
Ok, this seems fair in terms of CSS. I like that you're using the $pt vars. Are you sure about the mixed _ and -- convention for CSS class names?